packages/valko-ui-docs/src/pages/docs/ui/MenuPage.vue

```diff
@@ -1,3 +1,395 @@
+    values: 'true, false',
+    default: ''
+  },
+  {
+    prop: 'onClick',
+    required: false,
+    description: 'Emit for the Item.',
+    values: '() => void',
+    default: ''
+  }
+]
+
+const menuEmits = [
+  {
+    event: 'itemClick',
+    description: 'Emitted when an item in the Menu is clicked.',
+    values: 'Item',
+    type: '(item: MenuItem) => void'
+  }
+]
+
+const menuSlots = [
+  {
+    name: 'default',
+    description: 'Slot for the main content of the menu item.',
+    example: '<template #default>\n  <p>This is the main content of the menu item.</p>\n</template>'
+  }
+]
+
+// active, onClick & menu generation for example-section
+
+const generateMenuItems = (items: { value: string; label: string }[], groupName: string) => {
+  return items.map(item => ({
+    key: item.value,
+    text: item.label,
+    group: groupName
+  }))
+}
+
+const activeItemsList = ref<Record<string, MenuItem['key'] | null>>({})
+
+const onItemClick = (item: MenuItem, menuKey: string) => {
+  activeItemsList.value[menuKey] = item.key
+}
+
+// active & onClick for playground menu
+
+const activeItem = ref<MenuItem['key'] | null>(null)
+
+const onClick = (item: MenuItem) => {
+  activeItem.value = item.key
+}
+
+onMounted(() => {
+  // Initialize every menu with the first item active
+  // Click the first item in each menu
+  colorOptions.forEach((_, index) => {
+    const firstItemKey = colorOptions[0].value
+    const menuKey = `color-menu-${index}`
+    activeItemsList.value[menuKey] = firstItemKey
+    const menuItem = menuItems.find(item => item.key === firstItemKey)
+    if (menuItem) {
+      onItemClick(menuItem, menuKey)
+    }
+  })
+
+  variantOptions.forEach((_, index) => {
+    const firstItemKey = variantOptions[0].value
+    const menuKey = `variant-menu-${index}`
+    activeItemsList.value[menuKey] = firstItemKey
+    const menuItem = menuItems.find(item => item.key === firstItemKey)
+    if (menuItem) {
+      onItemClick(menuItem, menuKey)
+    }
+  })
+
+  shapeOptions.forEach((_, index) => {
+    const firstItemKey = shapeOptions[0].value
+    const menuKey = `shape-menu-${index}`
+    activeItemsList.value[menuKey] = firstItemKey
+    const menuItem = menuItems.find(item => item.key === firstItemKey)
+    if (menuItem) {
+      onItemClick(menuItem, menuKey)
+    }
+  })
+
+  sizeOptions.forEach((_, index) => {
+    const firstItemKey = sizeOptions[0].value
+    const menuKey = `size-menu-${index}`
+    activeItemsList.value[menuKey] = firstItemKey
+    const menuItem = menuItems.find(item => item.key === firstItemKey)
+    if (menuItem) {
+      onItemClick(menuItem, menuKey)
+    }
+  })
+
+  // Click on the first item in the playground menu
+  const firstPlaygroundItemKey = menuItems[0].key
+  activeItem.value = firstPlaygroundItemKey
+  const firstPlaygroundItem = menuItems.find(item => item.key === firstPlaygroundItemKey)
+  if (firstPlaygroundItem) {
+    onClick(firstPlaygroundItem)
+  }
+})
```

Comment:
you don't need to do this click thing, just add `active: true` to the first menu item and that's it.